docs/docs/1-getting-started/4-crud-generator/1-api-generator.md

```diff
@@ -86,6 +86,28 @@ Now you can run the generator with `npm run api-generator`. The generated files
 Although this is generated code, it should still be checked into the repository. This enables a quick start of the API.
 :::
 
+### Watch Mode
+
+The api-generator script also supports the `-w` or `--watch` flag. This will watch for changes in the .entity.ts files and regenerate the corresponding files.
+
+```json
+{
+  ...
+  "scripts": {
+    "api-generator:watch": "rimraf 'src/*/generated' && comet api-generator --watch",
```

Comment:
This is outdated, it should be `comet-api-generator generate`

Reply:
updated.